calcom/cal.com
# feat: add range of dates for availability over-ride

## What does this PR do?

This PR adds the functionality to select a range of dates when selecting the overrides for availability.

Fixes #7909
/claim #7909 

https://github.com/calcom/cal.com/assets/47187878/6773e238-d071-4ca2-bc8b-93129e3735bf

## Type of change


## How should this be tested?

- Go To the availabilities tab
- Click on add override
- Now click on 2 separate dates and check that all the dates in between got selected
- Add the override
- save it and see that the override is applied to the availabilities

## Mandatory Tasks


## Technical Details

- Have changed the DatePickers selected param to also accept an array of dates 
- This array is used for selecting a range of dates
- Have omitted to select a range of dates when editing because it made sense that a user only changes the time range or switches to another single date when editing a single entity.
